test/integration/http_benchmark.cc

```diff
@@ -0,0 +1,136 @@
+#include <string>
```

Comment:
I think this would be best split out as a distinct PR. Also, more comments explaining what is being measured, how to use this benchmark, etc. would make sense.

Reply:
Removed for now.